tensorflow_addons/layers/normalizations.py

```diff
@@ -350,10 +351,8 @@ class FilterResponseNormalization(tf.keras.layers.Layer):
     Input shape
         Arbitrary. Use the keyword argument `input_shape`
         (tuple of integers, does not include the samples axis)
-        when using this layer as the first layer in a model. This layer, as of now,
-        works on a 4-D tensor where the tensor should have the shape [N X H X W X C]
-
-        TODO: Add support for NCHW data format and FC layers.
+        when using this layer as the first layer in a model. This layer supports
+        arbitrary tensors.
```

Comment:
Mention the assumptions made: 
1. Excepts the input tensor to be at least 2D.
2. Excepts the batch dim to be `0` in all cases.